readme.md

```diff
@@ -75,7 +75,7 @@ watchedObject.a.b[0].c = true;
 
 ## API
 
-### onChange(object, onChange)
+### onChange(object, onChange, [isShallow])
```

Comment:
I think the third argument should be an options-object. It's more readable than a boolean-trap.

Reply:
See https://github.com/sindresorhus/log-update/blob/abd40b04b7dcfcc6dba91171ddaf0d7f271e7788/index.d.ts#L43-L58 for how to add it to the TS definition.